randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -41,11 +41,14 @@ Extra - room_id: [0]
   * Extra - door_idx: (79,)
   > Door to Entrance Recharge Room
       All of the following:
+          Screw Attack
           Any of the following:
+              # Break single bomb blocks
               Can Use Power Bombs
               Screw Attack and Knowledge (Beginner)
               Jump Bombjump (Expert) and Can Use Bombs
```

Comment:
is this easier with HJ?

Reply:
This path looks odd. Can you actually jbj to break those if you don't have hj?

Also there's no need to climb a single side at all for line 51.

Could also add Movement (beginner/intermediate) to only break the top 2 sa blocks to create a platform.

Reply:
> This path looks odd. Can you actually jbj to break those if you don't have hj?
just tried, dont think so, however
> Could also add Movement (beginner/intermediate) to only break the top 2 sa blocks to create a platform.
you could potentially get a NHJ JBJ by using this method, If anyone wants to try and get of vid of this